packages/api-client/src/main/http/BackendErrorMapper.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {UnknownConversationError} from '../conversation/';
+import {BackendError, BackendErrorLabel, StatusCode} from '../http/';
+import {UnknownUserError} from '../user/';
+
+class BackendErrorMapper {
+  public static mapBackendError(error: BackendError): Error {
+    const errors: {
```

Comment:
We could probably extract this config object outside the scope of the method, so that it's not created at every call of the function